### DISCUSSION TITLE
blog/nested-route/introducing-multi-part-posts-with-nested-routing

### DISCUSSION THREAD

**Discussion by giscus** [svandegar/seraphin#1] [Announcements]
# blog/nested-route/introducing-multi-part-posts-with-nested-routing

The blog template supports posts in nested sub-folders. This can be used to group posts of similar content e.g. a multi-part course. This post is itself an example of a nested route!

http://localhost:3000/blog/nested-route/introducing-multi-part-posts-with-nested-routing

**Comment by svandegar** [comment]
Hello!
